sys/netbsd/lwp.txt

```diff
@@ -28,6 +29,19 @@ _lwp_setname(target lwpid, name ptr[in, string])
 _lwp_getname(target lwpid, name ptr[out, string], size bytesize[name])
 _lwp_ctl(features int32, address ptr[in, ptr[in, lwpctl]])
 
+resource semid[intptr]
+
+_ksem_init(val int32, idp ptr[out, semid])
+_ksem_open(name buffer[in], oflag flags[semopen_flags], mode flags[open_mode], val int32, idp ptr[in, semid])
+_ksem_unlink(name buffer[in])
+_ksem_close(id semid)
+_ksem_post(id semid)
+_ksem_wait(id semid)
+_ksem_trywait(id semid)
+_ksem_getvalue(id semid, val ptr[out, int32])
+_ksem_destroy(id semid)
+_ksem_timedwait(id semid)
+
```

Comment:
Shouldn't this be `_ksem_trywait()` here?

We miss `_ksem_timedwait()`

Reply:
Sorry for missing out on this. The recent commit takes it into notice.